internal/terraform/terraform_test.go

```diff
@@ -92,7 +92,10 @@ func testModuleWithSnapshot(t *testing.T, name string) (*configs.Config, *config
 func testModuleInline(t testing.TB, sources map[string]string) *configs.Config {
 	t.Helper()
 
-	cfgPath := t.TempDir()
+	cfgPath, err := filepath.EvalSymlinks(t.TempDir())
```

Comment:
off-topic -- wondering what necessitated this? Asking bc of the `EvalSymlinks` on Windows problems, so trying to figure out where the legitimate uses are.

Reply:
It's somewhat annoying but the easiest way of making tests pass for me.

Specifically, some logic inside of `Plan()` and `Validate()` evaluates symlinks and so if there are symlinks in the way, we end up with different paths than those that come from `t.TempDir()`.

In my case `t.TempDir()` points somewhere to `/var/tmp/...` whereas `/var` is a symlink to `/private/var`. I believe this is default on macOS.